web/src/lib/components/nostr/ReplyNote.svelte

```diff
@@ -3,9 +3,19 @@
     import Nip05Checkmark from "$lib/components/nostr/Nip05Checkmark.svelte";
 
     export let message;
+    export let onReply = null;
+    export let closeButton: boolean = false;
 </script>
 
 <div class="flex my-4">
+    {#if closeButton}
+        <div class="card-actions justify-end">
+            <button class="btn btn-square btn-sm tooltip flex" data-tip="Close reply message" on:click|preventDefault={() => onReply(null)}>
```

Comment:
if onReply is null, how is it called here? The IDE should show you an error.

I think either you make it non-nullable, by initializing it with () => {}, or check for null here when you call it...

Reply:
@ibz for the `onReply`, I'm doing just `export let onReply;`. My IDE is not showing problems. Maybe he knows that this component will be mounted always passing the parameter from the other component?

Anyways, I can change it to `() => {}`.